scripts/build-documentation.sh

```diff
@@ -18,8 +18,9 @@ COMMIT_ID="$(git rev-parse HEAD)"
 TEMP=$((git describe --exact-match $COMMIT_ID) 2>&1)
 
 if [[ $TEMP != *"fatal"* ]]; then
-    echo "New tag detected; Uploading the documentation to opae.github.io"
-    /bin/bash scripts/push-documentation.sh 
+    echo "New tag detected; Uploading documentation under new tag on opae.github.io"
+    /bin/bash ../scripts/push-documentation.sh new_tag
 else
-    echo "No new tag detected; Documentation will not be uploaded to opae.github.io"
+    echo "No new tag detected; Latest documentation will be uploaded to opae.github.io"
```

Comment:
Is this right? E.g. what happens on pull request builds (without tags) - they shouldn't update "latest", should they?

Reply:
This is because we don't have a daily or a weekly build on travis. Should this be added in some sort of scheduled build?

Reply:
The script now uploads the documents from master branch instead of the PR branch if no tag is detected. That way, "latest" will always have the docs version that is present on master.